### ISSUE TITLE
[Oculus Quest] The UI remains frozen when trying to open a downloaded file with another app

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#3677]
**[Affected Versions]:**
- Firefox Reality version 11 RC7 (45f818a).

**[Affected Platforms]:** 
- Oculus Quest - Android 7.1.1

**[Prerequisites]:**
- The Firefox Reality browser is open.
- Have a '.tiff' image downloaded (I used this [page](https://file-examples.com/index.php/sample-images-download/sample-tiff-download/)).

**[Steps to reproduce]:**
1. Open the Downloads library and open the .tiff image.
2. Click the "Open" button from the prompt (open with external app) displayed.
3. Observe the behavior.

**[Expected results]:**
- The file is correctly opened with the external app.

**[Actual results]:**
- The FxR UI remains frozen.

**[Notes]:**
- This is the only example I really know that has an external app to be opened with and that's why I set the severity to "Normal". But, I recommend fixing this issue in the near releases when maybe other apps for different formats will be available. 
- The "Gallery" seems to be displayed in the universal menu if the Oculus button is pressed.
- Attached is a screen recording of the issue:
	- Screen Recording: [link](https://drive.google.com/file/d/15zPENLxheE2N8-dvKuOzO79Xifb4IRp0/view?usp=sharing).

**Comment by keianhzo** [comment]
@Softvision-RemusDranca I can't reproduce the freezing. The gallery opens and I get a black screen but FxR remains ok. 

The same seems to work ok in Pico, but Oculus seems to have a specific gallery app which apparently is not the default one that the system calls when starting the activity with the implicit intent for the given uri and image content type.

Most of the headsets don't seem to either have enough apps to handle most of the content that the browser can download and doesn't support natively so we might need to consider removing the "Open with External App" support.

**Comment by Softvision-RemusDranca** [comment]
@keianhzo I've retested the issue using RC 10, v11, and now I get the black screen as well, as it did in the past. I marked it as a regression because of the behavior I encountered in RC7 where the UI was frozen, like it can be observed in the video and also, I thought that it could have been affected by the new Oculus update. 
Considering this I will remove the "regression" label, and leave it open so discussions can be continued on your idea of removing the "Open with External App" support. Thanks!
